packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -319,14 +319,6 @@ class IOSSimulator extends Device {
     // Prepare launch arguments.
     final List<String> args = <String>['--enable-dart-profiling'];
 
-    if (!prebuiltApplication) {
-      args.addAll(<String>[
-        '--flutter-assets-dir=${fs.path.absolute(getAssetBuildDirectory())}',
-        '--dart-main=${fs.path.absolute(mainPath)}${debuggingOptions.buildInfo.previewDart2?".dill":""}',
-        '--packages=${fs.path.absolute('.packages')}',
```

Comment:
Is `--packages` not needed anymore either?

/cc @aam

Reply:
`--packages` only made sense if supplying dart code. This argument is already ignored if passed a .dill file.